### PR TITLE
Adding flex grow to card wrapper

### LINKED ISSUE #28726: Step previews in notebook editor are not working
**Describe the bug**
The question preview shows a blank table

**Logs**
no error logs from both console log and backend

**To Reproduce**
1. New question on any table
2. Click on the :arrow_forward:  button to preview the table

**Expected behavior**
Should show the table result

**Screenshots/videos**
image

**Information about your Metabase Installation:**
current master(ee44d788ea2bad9a491aa93528cbbba4516ef3d6)

**Severity**
Regression

### PR DESCRIPTION
**Pull request by npfitz**
Fixes #28726 
Reproduces #28726 

Issue was caused by the preview not being given a proper height. We recently added a wrapper around the `CardVisualization` component, but the parent component for previews assumes that it's direct child will have `flex-grow` applied to use up the space.

How to test:
- Start a new question
- Sample dataset -> Orders
- Click the preview button, you should see a preview as a table.

image

### CODE REVIEW

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 1]
```diff
@@ -550,6 +550,18 @@ describe("scenarios > question > notebook", () => {
       });
     });
   });
+
+  it("should render previews of questions (#28726)", () => {
+    startNewQuestion();
+    cy.findByText("Sample Database").click();
+    cy.findByText("Orders").click();
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:557]:
```suggestion
    openOrdersTable({ mode: "notebook" });
```

**Review by alxnddr — APPROVED** [review]
@npfitz thanks for the fix

**Review by nemanjaglumac — CHANGES_REQUESTED** [review]
E2E test failures are legit.
This approach introduces more regressions.

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 1 above)
Comment (npfitz) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js]: Ohh! Perfect! Thanks @nemanjaglumac

**Review by nemanjaglumac — APPROVED** [review]
Tests are now passing after @alxnddr 's change.
But the chance of this introducing a UI regression somewhere else in code is not insignificant.

Let's keep an eye on it.

File: frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx [hunk 2]
```diff
@@ -484,7 +484,7 @@ class Visualization extends React.PureComponent {
           ) : (
             <div
               data-card-key={getCardKey(series[0].card)}
-              className="flex flex-column full-height"
+              className="flex flex-column flex-full"
```
Comment (iethree) [inline frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: this change broke internal scroll for object detail viz in the query builder and on dashboards 😢 

Before | After
--- | ---
scrolldetail | noScroll

Comment (npfitz) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: Well... That's not ideal :/

Comment (npfitz) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: Does adding back the `full-height` class bring back the scroll functionality?

Comment (iethree) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: yeah, having both classes seems to work for this situation

Screen Shot 2023-03-07 at 10 16 13 AM

Comment (npfitz) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: Well damn. I think having both classes made cards not render properly

Comment (iethree) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: oh wait. it fixes the scrollbar issue, but it cutz of the paginator control.

Screen Shot 2023-03-07 at 10 18 22 AM

Comment (npfitz) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: Might need to get a bit more creative with a fix then. @calherries actually had a solution where he moved the wrapper up a couple levels. Not sure if that approach would tick all our boxes or not.

https://github.com/metabase/metabase/compare/fix-vis-wrapper

Comment (calherries) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: You could try that approach, but the downside is the visualization header is included in the download png. I didn't check whether that was acceptable 
https://github.com/metabase/metabase/pull/28546

Comment (npfitz) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: We could also just make adjustments to the preview container

Comment (iethree) [reply frontend/src/metabase/visualizations/components/Visualization/Visualization.jsx:487]: I haven't looked at it as much as you, but the preview container sounds like the better place to look than messing with `Visualization` at all

**Pull request merged**